libs/service-portal/information/src/lib/paths.ts

```diff
@@ -0,0 +1,12 @@
+export enum InformationPaths {
+  Child = '/min-gogn/barn/:nationalId',
+  FamilyMember = '/min-gogn/:nationalId',
+  Spouse = '/min-gogn/maki/:nationalId',
+  MyInfoRoot = '/min-gogn',
+  UserInfo = '/min-gogn/minar-upplysingar',
+  Endorsements = '/min-gogn/medmaeli',
+  Company = '/fyrirtaeki',
+
+  Petitions = '/medmaeli',
+  PetitionsAdminView = '/medmaeli-admin',
```

Comment:
duplication of what is listed in Endorsements/.../paths ?

Reply:
🙈 yeah this should be imported from `/endorsements` probably